wear/src/main/java/io/homeassistant/companion/android/home/views/SensorUi.kt

```diff
@@ -95,10 +101,48 @@ fun SensorUi(
 private fun PreviewSensorUI() {
     val context = LocalContext.current
     CompositionLocalProvider {
-        SensorUi(
-            sensor = null,
-            manager = batterySensorManager,
-            basicSensor = runBlocking { batterySensorManager.getAvailableSensors(context).first() }
-        ) { _, _ -> }
+        ThemeLazyColumn {
+            item {
+                SensorUi(
+                    sensor = Sensor(
+                        "battery_level",
+                        0,
+                        true,
+                        state = "80",
+                        unitOfMeasurement = "%"
+                    ),
+                    manager = batterySensorManager,
+                    basicSensor = runBlocking {
+                        batterySensorManager.getAvailableSensors(context).first()
```

Comment:
For the previews, maybe always get the sensor by id instead of relying on `first()`/`last()`? Otherwise it might break if the sensor order ever changes. You could also move the `batterySensorManager.getAvailableSensors(context)` call out of the inidividual items so it can be reused.

Reply:
ok done :)

Reply:
Now `batterySensors.first { ... }` is not using suspending functions so `runBlocking` can be removed ;)